ansible_collections/arista/avd/molecule/eos_designs_negative_unit_tests/inventory/host_vars/duplicate-interfaces-core-interfaces-with-port-channel.yml

```diff
@@ -33,6 +33,7 @@ core_interfaces:
       interfaces: [ Ethernet1, Ethernet2 ]
       profile: isis_bb_profile
       ip: ["unnumbered loopback0", "unnumbered loopback0"]
+      as: [65000, 65002]
```

Comment:
why do we need to add AS and IP info here?

Reply:
We were getting the below errors and molecule was failing

```
{"changed": false, "msg": "core_interfaces.p2p_links.[].as or core_interfaces.p2p_links_profiles.[].as”}
{"changed": false, "msg": "core_interfaces.p2p_links.[].ip, .subnet or .ip_pool"}
```